src/main/resources/errors.yaml

```diff
@@ -202,6 +202,20 @@ request-errors:
       
       ${SNIPPET.INEFFICIENT_QUERY}
 
+
+  - scope: WARNING
+    code: COMPARISON_FILTER_UNSUPPORTED_BY_INDEXING
+    title: Use of $lt/$gt/$lte/$gte (comparison filter) on indexed columns
```

Comment:
nit: can we use " $lt, $gt, $lte or $gte " in the title and body pls, it is easier to read

Reply:
fixed, thanks